src/PresentationalComponents/Inventory/InventoryDetails.js

```diff
@@ -46,7 +60,7 @@ const InventoryDetails = ({ entity }) => {
     >
       <PageHeader className="pf-m-light ins-inventory-detail">
         {entity && <Breadcrumbs current={entity.display_name || entity.id} />}
-        <InventoryDetailHead hideBack fallback="" />
+        <InventoryDetailHead hideBack fallback={<InventoryHeadFallback />} />
```

Comment:
this loading state seems to come from a different place in inventory (tho it's hard to pinpoint where, because of minification)
![Screenshot from 2024-10-30 11-52-16](https://github.com/user-attachments/assets/8f8e7115-1bdd-4a8e-a8d3-71da24373058)
![Screenshot from 2024-10-30 11-52-33](https://github.com/user-attachments/assets/27c5c44a-ab5d-46cf-a2e2-28ce49c0b007)